src/lib/gundb/UserPropertiesClass.js

```diff
@@ -161,10 +163,12 @@ export default class UserProperties {
    * Reset properties to the default state
    */
   async reset() {
-    const { defaultProperties } = UserProperties
+    const { data } = this
+    const { defaultProperties, keepFields } = UserProperties
+    const dataBeenReset = assign({}, defaultProperties, pick(data || {}, keepFields))
```

Comment:
keep some data (e.g. lastInviteState) when account removed, to avoid invitees numbers displayed again once account re created. That's actually nothing personal and private so we could do it